atlassian_jwt_auth/contrib/tests/utils.py

```diff
@@ -1,20 +1,21 @@
 import atlassian_jwt_auth
 
 
-class StaticPublicKeyRetriever(object):
```

Comment:
I am not disagreeing with that part. I am asking why we need this code churn. You need not move the `StaticPublicKeyRetriever` class under the `get_static_retriever_class` class and in fact I personally disagree with doing so.